telegram/ext/filters.py

```diff
@@ -1871,15 +1875,57 @@ def filter(self, message: Message) -> bool:
     """
 
 
-class _Sticker(MessageFilter):
+class Sticker:
+    """Filters messages which contain a sticker.
+
+    Examples:
+        Use this filter like: ``filters.Sticker.VIDEO``. Or, just use ``filters.Sticker.ALL`` for
+        any type of sticker.
+
+    Caution:
+        ``filters.STICKER`` itself is *not* a filter.
+    """
+
     __slots__ = ()
 
-    def filter(self, message: Message) -> bool:
-        return bool(message.sticker)
+    class _All(MessageFilter):
+        __slots__ = ()
 
+        def filter(self, message: Message) -> bool:
+            return bool(message.sticker)
+
+    ALL = _All(name="filters.Sticker.ALL")
+    """Messages that contain :attr:`telegram.Message.sticker`."""
+
+    class _Animated(MessageFilter):
+        __slots__ = ()
+
+        def filter(self, message: Message) -> bool:
+            return bool(message.sticker) and bool(message.sticker.is_animated)  # type: ignore
+
+    ANIMATED = _Animated(name="filters.Sticker.ANIMATED")
+    """Messages that contain :attr:`telegram.Message.sticker` and is an animated sticker."""
+
```

Comment:
maybe explicitly reference `Sticker.is_animated` here? same below for video/static